extensions/cal-com-share-meeting-links/package.json

```diff
@@ -19,11 +19,18 @@
   ],
   "commands": [
     {
-      "name": "index",
+      "name": "share-meeting-link",
```

Comment:
Do you mind we keep the name `index` for this? I know it would be more nice to change but this name is the unique ID where alias, hotkeys and ranking is stored 🙂

Reply:
No problem ☺️